BlazarService/src/test/java/com/hubspot/blazar/service/GitBranchUpdaterTest.java

```diff
@@ -0,0 +1,117 @@
+package com.hubspot.blazar.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.jukito.JukitoRunner;
+import org.jukito.UseModules;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kohsuke.github.BlazarGHRepository;
+import org.kohsuke.github.BlazarGHUser;
+import org.kohsuke.github.GHRepository;
+
+import com.google.common.base.Optional;
+import com.google.inject.Inject;
+import com.hubspot.blazar.BlazarServiceTestBase;
+import com.hubspot.blazar.BlazarServiceTestModule;
+import com.hubspot.blazar.base.GitInfo;
+import com.hubspot.blazar.command.GitBranchUpdater;
+import com.hubspot.blazar.config.BlazarConfiguration;
+import com.hubspot.blazar.data.service.BranchService;
+import com.hubspot.blazar.util.GitHubHelper;
+
+import io.dropwizard.db.ManagedDataSource;
+
+
+@RunWith(JukitoRunner.class)
+@UseModules({BlazarServiceTestModule.class})
+public class GitBranchUpdaterTest extends BlazarServiceTestBase {
+  @Inject
+  private BranchService branchService;
+  @Inject
+  private BlazarConfiguration blazarConfiguration;
+  @Inject
+  private GitHubHelper gitHubHelper;
+
+  @Inject
+  @Before
+  public void before(ManagedDataSource dataSource) throws Exception {
+    runSql(dataSource, "InterProjectData.sql");
+  }
+
+  @Test
+  public void doesNotChangeUnchangedBranch() {
+    GitInfo unchangedBranch = branchService.get(1).get();
+    buildUpdater(unchangedBranch).run();
+    GitInfo maybeChangedBranch = branchService.get(1).get();
+    assertThat(maybeChangedBranch).isEqualTo(unchangedBranch);
+  }
+
+  @Test
+  public void deletesBranchWithoutHost() {
+    GitInfo branchWithoutConfiguredHost = buildGitInfo("git.bar.example.com/org/repo#branch");
+    branchWithoutConfiguredHost = branchService.upsert(branchWithoutConfiguredHost);
+    GitInfo unchangedBranch = branchService.get(1).get();
+    assertThat(branchWithoutConfiguredHost.isActive()).isTrue();
+
+    buildUpdater(unchangedBranch).run();
```

Comment:
Your have tested that in the previous test. Why do you want to test it again here?